### ISSUE TITLE
Knife azure server create issue after upgrading chefdk version

### ISSUE THREAD

**Issue by rahulkumardbit** [chef/knife-azure#448]
I have recently upgraded my Chefdk version from 0.10.0 to 1.5.0 and since then I am getting this error
``
....Waiting for virtual machine to reach status 'provisioning'....ERROR: Failed to open TCP connection to management.cor
e.windows.net:443 (A connection attempt failed because the connected party did not properly respond after a period of ti
me, or established connection failed because connected host has failed to respond. - connect(2) for "management.core.win
dows.net" port 443)
``

knife command:

``
knife azure server create --azure-dns-name 'dnsname' --azure-service-location 'East US' --azure-source-image '5112500ae3b842c8b9c604889f8753c3__OpenLogic-CentOS-65-20160309' --azure-vm-name 'vmname' --azure-vm-size 'ExtraLarge' --ssh-user 'azureuser' --ssh-password 'xxxxxxx' --tcp-endpoints '80,8080,443,5432,3306,8085'
``

Chefdk version: 1.5.0
OS: Windows 7

**Comment by harikesh-kolekar** [comment]
@rahulkumardbit , Can't reproduce the issue I have tried by the following command 
```

knife azure server create --azure-dns-name 'harrytest5' --azure-service-location 'East US' --azure-source-image '5112500ae3b842c8b9c604889f8753c3__OpenLogic-CentOS-65-20160309' --azure-vm-name 'harrytest5' --azure-vm-size ' ExtraLarge' --ssh-user 'azureuser' --ssh-password 'Azure@123456' --tcp-endpoints '80,8080,443,5432,3306,8085' -VV
```
Chefdk version: 1.5.0
`Can you please share log with -VV option?`

**Comment by NimishaS** [comment]
Fixed in `knife-azure 1.8.0`.
